### DISCUSSION TITLE
Sysmon logs not showing in kibana with wazuh shipping

### DISCUSSION THREAD

**Discussion by Mastadamus** [Security-Onion-Solutions/securityonion#2964] [Unsupported Versions] [answered]
Ive set up wazuh and installed sysmon basically identical to this reddit posts similiar situation. Not getting logs in kibana. Did this ever get resolved? im not sure if i screwed up or if its just not working yet.
1. I put this in the ossec.conf file 
<localfile>
   <location>Microsoft-Windows-Sysmon/Operational</location>
   <log_format>eventchannel</log_format>
</localfile>
2. I have all others aspects of wazuh working. im getting alerts etc.
3. installed sysmon and its running alongside the recommended configuration.xml

https://www.reddit.com/r/securityonion/comments/iyx46c/no_sysmon_logs_in_kibana/
---------------------From reddit post.----------------
I am in the process of fixing this at the moment. The archives.json log file should contain the Sysmon logs if you look there (/nsm/wazuh/logs/archives/archives.json). The issue is that Filebeat is not picking up the archives.json, but the alerts.json, which only contains alerts. Furthermore, there is some additional work with the pipeline that likely needs to be completely for this to work correctly. Apologies, but this should be resolved by the next release.

**Comment by Mastadamus** [comment, marked as answer]
This is resolved. Even though i already accomplished this step, I stopped and restarted the wazuh agent and it started sending sysmon logs. I am now gtg.

**Reply by facyber** [reply]
Hey @Mastadamus yes this was resolved later and I've tested it and it worked. Could you tell me did you made any changes to `local_rules.xml `file? In official Wazuh documentation they say to add this bellow, which I added now in my new setup (but didn't do it in old setup), so not sure if it is important.

```
<group name="sysmon,">
 <rule id="255000" level="12">
 <if_group>sysmon_event1</if_group>
 <field name="sysmon.image">\\powershell.exe||\\.ps1||\\.ps2</field>
 <description>Sysmon - Event 1: Bad exe: $(sysmon.image)</description>
 <group>sysmon_event1,powershell_execution,</group>
 </rule>
</group>
```

Cheers

**Comment by Mastadamus** [comment]
Facyber,

I did not make those changes. The only think I did was add the location of the sysmon logs to the ossec.conf file and restart the agent. I am pretty sure I restarted the agent initially, but for some reason it started functioning after a second start and restart so it may have been I simply forgot to earlier.
